docs/Earn/liquid-staking/dot/dot-liquid-staking-mechanics.md

```diff
@@ -0,0 +1,183 @@
+
+Polkadot deposit address:
+
+[ANKR deposit address](https://polkadot.subscan.io/account/11XAoN3fgVLJUmK4Ty2q2H5dGYYzRZe82F247SyzfwZbHpV) — address that receives the staked DOTs on Polkadot.
+
+Ethereum:
+
+[aDOTb](https://etherscan.io/address/0x5cc56c266143f29a5054b9ae07f3ac3513a7965e) — contract on the Ethereum network that mints or burns aDOTb tokens for the user in 1:1 ratio with the staked amount.
+
+[PolkadotPool](https://etherscan.io/address/0x59f767EC659E9FE01ebCf930465E2aD4Cc0F208e) — contract on the Ethereum network where the user sends their request to mint or burn aDOTb.
+
+## Staking Flow
+
+The flow consists of two parts: staking DOT and claiming aDOTb.
+
+### Staking DOT
+
+The workflow is both user- and ANKR-driven. The user part is Step 1 through 3, while the ANKR part is Step 3 through 5. User interacts with ANKR and Polkadot via the ANKR frontend. 
+
+Due to Polkadot backend architecture, we currently do not disclose our API endpoints to avoid scams or unregulated heavy traffic to the ANKR backend. 
+
+1. User requests the ANKR deposit address via the ANKR backend.
+
+2. User sends DOTs to the obtained ANKR deposit address via the ANKR backend.
+
+3. User sends their extrinsic id to the ANKR backend.
+
+4. Once the transaction has been finalized, the user can see their DOT balance on the StakeFi page the stake was made from.
+
+5. ANKR backend monitor deposits to the ANKR deposit address. Once new DOTs are available for staking, the backend:
+   1. Transfer the DOTs to one of our treasury accounts if necessary. 
+   2. Bond DOTs for staking
+   3. Constantly nominate a subset of validators.
+
+### Claiming aDOTb
+
+The workflow is user-driven. User interacts with ANKR and Polkadot via ANK frontend. 
+
+1. User sees their claimable balance on the DOT staking dashboard. The balance is obtained from the ANKR backend; the user `address` is extracted from their Polkadot.js wallet; `claimable` is the amount that can be currently claimed and is an aggregate of the stake and reward.
+
+2. User sees their staking history on the DOT staking dashboard. 
+
+3. For claimable aDOTb, the user sees a suggestion to claim them. In order to claim, the user must unlock and connect their Metamask or another compatible Ethereum wallet to the Ethereum network.
+
+4. When claiming, the user sees the claming form and inputs the amount of aDOTb to be claimed along with the user’s Ethereum address. By default, ANKR suggests the max possible claim and address from the connected user’s Ethereum wallet.  
+   Upon claim confirmation, the user interacts with Polkadot.js extension to send a request to the ANKR backend to get a notarization signature. To send the request, the user needs to include a signature in it. The signature is created by signing the marshalled message `<Bytes>StakeFi Signed Message:\n[POLKADOT_PUBLIC_KEY_HEX]\n[ETHEREUM_ADDRESS]\n[CURRENT_TIMESTAMP]</Bytes>`. The resulting signature is included in the request.
```

Comment:
Is this what the user sees? They will be prompted to sign in the `Polkadot.js` extension wallet which will send the message to the back end. However, I don't think they see that message. Please check.

Reply:
rephrased. thx